### DISCUSSION TITLE
Scoring Clarification: Minimum distance between the ball and the tag

### DISCUSSION THREAD

**Discussion by ishabhray** [larics/icuas22_competition#60] [Q&A] [answered]
Hello!
We wanted a clarification as to how the minimum distance between the ball and the center of the tag is calculated. Is it calculated from the point where the ball hits the wall or from the point which has the least Euclidean distance (not necessarily on the wall)?

Thanks!!

**Comment by fpetric** [comment, marked as answer]
Hi @ishabhray, thanks for asking. 

the distance is calculated as the smallest Euclidean distance of the ball to the tag from the moment the ball is launched until it lands on the floor.

**Reply by afaroo01** [reply]
Hello @fpetric, I wanted to ask, can you provide us the artag ground truth positions in the different arena from where you calculate the estimated distance from our detected published coordinates? We want to improve on our approach but couldn't find the artag location provided for each arena.

**Reply by fpetric** [reply]
We change the tile location when evaluating solutions, you can do the same using the session.yml file (export different TILE_X and so on). The positions specified in the session.yml file are only examples on how to specify the location of the tag, especially important is the orientation.

**Reply by afaroo01** [reply]
OK, will try it. many thanks

**Reply by h-utkuunlu** [reply]
> 

Hi there,

Do you take into account the ball's radius in computing the minimum distance from the marker? If not, then zero error is never possible.
